readme.txt

```diff
@@ -116,6 +116,57 @@ You'll find the [FAQ on Yoast.com](https://yoast.com/wordpress/plugins/seo/faq/)
 
 == Changelog ==
 
+= 3.4.0 =
+
+Release Date: July 19th, 2016
+
+* Enhancements:
+	* Adds the setting and user option to disable the SEO analysis.
+	* Adds the readability score to the post and term overview.
+	* Disables the analysis marker buttons when switching from visual to text view in the editor.
+	* Accessibility enhancements:
+		* Improves the headings in the dashboard widget.
+		* Improves the headings hierarchy on the following pages:
+			* titles and metas
+			* user profile
+			* advanced settings
+			* social settings
+			* XML sitemap
+			* general settings
+		* Improves the headings hierarchy for dashboard alerts.
+		* Improves the debug information headings.
+		* Adds a legend on the titles and metas page.
+		* Improves the intro text of the social tabs.
+		* Always use the buttons css class for buttons.
```

Comment:
This can be a little more user-friendly. Something like `Improves consistency of buttons using WordPress styles.`